### PR TITLE
Replace calls to keyval with expl3 functions

### PR DESCRIPTION
**Pull request by drehak**
Skeleton commit.

Partially resloves #96.

### CODE REVIEW

File: markdown.dtx [hunk 1]
```diff
@@ -25,11 +25,14 @@
...
-  keepaspectratio,
-}
+\ExplSyntaxOn
+\keys_set:nn { Gin }
+  {
+    width = \columnwidth,
+    height = 0.65\paperheight,
+    keepaspectratio,
+  }
+\ExplSyntaxOff
```
Comment (Witiko) [inline markdown.dtx:35]: `\setkeys{Gin}` configures the `graphicx` package. The graphicx package uses and will use `keyval`, i.e. this is not a useful test.

File: markdown.dtx [hunk 2]
```diff
@@ -14929,6 +14933,7 @@ Example themes provided with the Markdown package include:
 %    ```` tex
 %    \documentclass{article}
 %    \usepackage[theme=witiko/dot]{markdown}
+%    TODO replace with l3keys functions
```
Comment (Witiko) [inline markdown.dtx:14939]: `\setkeys{Gin}` configures the `graphicx` package. The graphicx package uses and will use `keyval`.

Comment (Witiko) [reply markdown.dtx:14939]:
```suggestion
```

File: markdown.dtx [hunk 3]
```diff
@@ -15002,6 +15007,7 @@ following content:
 ```` tex
 \documentclass{article}
 \usepackage[theme=witiko/dot]{markdown}
+% TODO replace with l3keys functions
```
Comment (Witiko) [inline markdown.dtx:15013]: As above.

Comment (Witiko) [reply markdown.dtx:15013]:
```suggestion
```

File: markdown.dtx (hunk 1 above)
Comment (drehak) [reply markdown.dtx:35]: Thanks, that explains a lot. :)

**Pull request closed without merging**